packages/experiments/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -90,13 +95,13 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>> extend
 
     return (
       <div
-        ref={ this._resolveRef('root') }
+        ref={ this.root }
         className={ css(
           'ms-BasePicker',
           className ? className : '') }
       >
         <FocusZone
-          ref={ this._resolveRef('focusZone') }
+          ref={ this.focusZone }
```

Comment:
should be componentRef

Reply:
So this is/was a bug?

Reply:
In that case should this be done everywhere? 

What is the difference in this case between regular ref and componentRef? For example we have this sometimes :P

```ts
<FocusZone
        { ...getNativeProps(this.props, divProperties) }
        direction={ FocusZoneDirection.horizontal }
        ref={ this._onRootRef }
        componentRef={ this._focusZone }
```